webapp-django/crashstats/crashstats/views.py

```diff
@@ -31,6 +31,17 @@
 datetime.datetime.strptime('2013-07-15 10:00:00', '%Y-%m-%d %H:%M:%S')
 
 
+def ratelimit_blocked(request, exception):
+    # http://tools.ietf.org/html/rfc6585#page-3
+    status = 429
+    if request.path.startswith('/api/'):
+        # return a super spartan message
+        # we could also do something like `{"error": "Too Many Requests"}`
```

Comment:
Yeah, it's confusing. I wonder what GitHub does. Whatever they do we ought to do :)